### DISCUSSION TITLE
Automated download of validation results from MS

### DISCUSSION THREAD

**Discussion by freddydk** [microsoft/AL-Go#116] [Ideas]
For AppSource Apps or PTEs which are deployed to customers, Microsoft will perform validation on these to check whether they are ready for next major etc.
We could add a workflow, which runs on a schedule and downloads the results of these validations and if my app is not compatible, create an issue in the repository, which would be flagged for people working on the app.

**Comment by BW-PA** [comment]
Would love this!  We currently have very manual processes around receiving the notification emails from Microsoft and then creating a DevOps ticket for them, cross checking if we already have one for the same extension installed at multiple customers.

If the workflow was run on a schedule would we still get notified only within the 2 weeks ahead of upgrade or would it be available anytime?  I guess I'm asking at what point in the timeline does MS perform their validation checks prior to Upgrades. 

And I can see some crossover with scheduled builds against nextMajor and nextMinor.  Though those processes don't currently have automated issue creation in the repository.  And probably shouldn't since the chances of nextMajor changing are higher the further away from release it is.
